### PR TITLE
Feature/main/aply ticker trust effect

### PR DESCRIPTION
**Pull request by jasc66**
# Description

se utiliza este ejemplo https://codepen.io/adamf321/pen/ExjmJrY y se aplica al codigo, se realizan modificaciones para que se adapte, se debe mejorar el efecto. 

[To obtain the URL, navigate to the ticket on Trello and click on the "Share" button located at the bottom right corner.
Paste between the parentheses: [Trello Ticket URL]()](https://trello.com/c/TrNYLGxF/27-check-these-tickers-examples)

# Checklist
Please ensure that you have completed the following tasks:

- [ ] I have performed a self-review of my own code
- [ ] I have run ``` npm run lint ``` to automatically fix any linting issues
- [ ] I have resolved all linting issues within the files modified in this PR
- [ ] I have provided visual evidence of the changes (e.g., screenshots, GIFs, or videos)
- [ ] I have added comments to my code, particularly in difficult-to-understand areas
- [ ] I have made corresponding updates to the documentation, if necessary
- [ ] My changes generate no new warnings in the terminal or browser console

# Visual Evidence
image

image

# Test Coverage (Optional)
If applicable, please attach screenshots of the test coverage report:

Run npm run ``` test:coverage:changed ``` to generate the report
Note: Including test coverage is optional for now, but highly encouraged. (This is a task we will implement soon).

### CODE REVIEW

File: components/Features/SingleFeature.tsx [hunk 1]
```diff
@@ -1,20 +1,30 @@
+'use client';
+import { useState, useEffect } from 'react';
```
Comment (m-ret) [inline components/Features/SingleFeature.tsx]: Un espacio acá.

File: styles/index.css [hunk 2]
```diff
@@ -31,3 +31,27 @@
...
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+  }
+  .move-right {
+    transform: rotate(-20deg);
+  }
+  
+  .move-left {
+    transform: rotate(20deg);
+  }
```
Comment (m-ret) [inline styles/index.css:57]: La indentación está mal. Es así:
```css
Copy code
@keyframes ticker-kf {
  0% {
    transform: translate3d(0, 0, 0);
  }
  100% {
    transform: translate3d(-108rem, 0, 0);
  }
}

.img-ticker {
  animation-name: ticker-kf;
  animation-duration: 18s;
  animation-timing-function: linear;
  animation-iteration-count: infinite;
}

.move-right {
  transform: rotate(-20deg);
}

.move-left {
  transform: rotate(20deg);
}
```

**Review by m-ret — CHANGES_REQUESTED** [review]

File: styles/index.css (hunk 2 above)
Comment (jasc66) [reply styles/index.css:57]: se aplica el cambio

**Comment by jasc66** [comment]
se ciera PR para dejar solo uno abierto

**Pull request closed without merging**